src/roiextractors/extraction_tools.py

```diff
@@ -461,6 +591,20 @@ def write_to_h5_dataset_format(
 
 # TODO will be moved eventually, but for now it's very handy :)
 def show_video(imaging, ax=None):
+    """Show video as animation.
+
+    Parameters
+    ----------
+    imaging: ImagingExtractor
+        The imaging extractor object to be saved in the .h5 file
+    ax: matplotlib axis
+        Axis to plot the video. If None, a new axis is created.
+
+    Returns
+    -------
+    anim: matplotlib.animation.FuncAnimation
```

Comment:
I find it interesting (again, pseudo-standard stuff) that the name of the return variable is a part of the docstring

Not against it, since it can be a 'suggestion' used to give downstream implementations a more standard naming convention

But it also syntactically doesn't matter what name the variable was at time of return; only the expected type matters

Reply:
Yeah, technically, it doesn't matter in the code (in the sense that you can return a named variable or some expression), but it is just easier to describe/document the return when it has a name.  I also prefer named returns in general (with the exception of very short functions) because they are more self-commenting than return expressions (although I couldn't find any authoritative expression of this preference ex. StackOverflow).

Reply:
> but it is just easier to describe/document the return when it has a name.

I would agree with you if and only if the name of the return variable is nicely readable

In this case, a return name of `anim` is not verbose enough; on first glance I thought it was short for `animal`, and only upon reading the annotation type was it clear to be short for `animation`

Maybe another follow-up could be to choose better variable names for such things?

Reply:
See #238.